src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+body {
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+form:first-child {
+  margin: 20px 0;
+}
+
+.search-input {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: url("./images/Search.svg") no-repeat 26px 25px;
+  background-color: #fff;
+  background-size: 19px 19px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-input::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-input:focus {
+  font-family: "Avenir", sans-serif;
+  font-weight: 800;
+  background: url("./images/Search.svg") no-repeat 26px 25px linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
It doesn't work:
![image](https://user-images.githubusercontent.com/86923089/189735809-d4174e4d-0ff1-4d35-b342-1a3297d38041.png)

Don't set search image as background (you can use `absolute` positioning)